_entries/2016-05-04-hyperlinks.md

```diff
@@ -35,16 +35,20 @@ Not this
 
 Don’t use meaningless terms such as ‘click here’, ‘read more’ or ‘useful links’.
 
-### Drop the http://
+### Drop the https://
 
-When referencing a website, drop the `http://` prefix.
+You don't need to write `http://` or `https://` when referencing a website.
+
+If the site has a URL route without `www` you don't need to write it when you reference it. But keep the `www` if there is only a redirect in place.
```

Comment:
Responding to your longer comment below @joahua I don't quite follow - how could the current wording lead to a broken link?

Agreed on the second point about checking the URL, and maybe it's worth going a step further to also say it's bad to link to stuff behind a wall? eg `Make sure the link works for the user. Don't link from public content to something you need to sign in to see.`